install-and-configure/install/multi-cluster/secondary-clusters.md

```diff
@@ -12,7 +12,7 @@ See the [Additional resources](/install-and-configure/install/multi-cluster/seco
 
 ## Kubecost global
 
-Disable product caching and reduce query concurrency with the following parameters:
+Disable unnecessary containers/pods on secondary clusters.
```

Comment:
Are you recommending users to disable Kubecost containers/Pods here or other workloads? If the latter, why is this a suggestion? The verb "disable" in the context of Kubernetes also doesn't mean a whole lot; Pods are atomic and either exist or not. There is no present-but-disabled state. A nit, I realize, but as a user when I read this I don't quite understand.

Reply:
Good point about the verb "disable". But yes the interpretation is correct! We do not want to run pods such as forecating, grafana. And we do not want to run the cost-analyzer-frontend container. Thoughts on how to rephrase?

Reply:
I suggest we be prescriptive since we're the experts. Which Pods/containers can we tell users to disable? And how would they go about doing that? This is one valid example, I think, where a Helm values snippet is warranted in which we provide the prescription for a minimal secondary cluster configuration.

Reply:
In the code snippet below this text, we tell the user to set `--set federatedETL.agentOnly=true`. This is a Helm flag which when true will not render any of those extra pods/containers I listed above. Let me know what you think! Does something need to be more explicit?